frontend/src/routes/userLand/milestone/+page.svelte

```diff
@@ -199,8 +200,22 @@ const breadcrumbdata = $derived([
                                 class={` min-w-20 absolute top-0 left-0 w-full h-full object-cover transition duration-1000 ease-in-out ${imageIndex === currentImageIndex ? 'opacity-100' : 'opacity-0'}`}
                                 src={`${import.meta.env.VITE_MONDEY_API_URL}/static/m/${image.id}.webp`}
                                 alt=""
+                                on:click={() => {
+                                    modalImagePreviewOpen = true;
+                                }}
                         />
                     {/each}
+                    <Modal bind:open={modalImagePreviewOpen} size="xl" autoclose outsideclose>
+                        <div class="p-2">
```

Comment:
on (emulated) mobile I struggled to close this modal, as the close button was hidden under the navbar (due to its high z index) and clicking outside didn't seem to work. Do you see the same issue?

![image](https://github.com/user-attachments/assets/3ad15510-45df-4523-a40e-07b8476252c4)

Maybe we could also make this modal close when it is clicked on, e.g.?
```suggestion
                        <div class="p-2" on:click={()=>{modalImagePreviewOpen=false}}>
```